RvmSharp/RvmParser.cs

```diff
@@ -316,12 +324,15 @@ private static RvmNode ReadCntb(Stream stream)
             Console.WriteLine("--- Node with name " + name + " was a v4 node"); // TODO: Remove me when we have more controll of the v4 nodes. Currently may be related to "ARCHIV 1, but not known"
             Span<byte> bytes = stackalloc byte[4 * 7];
             stream.ReadExactly(bytes);
-            if (!bytes.SequenceEqual(ExpectedV4Bytes))
+
+            var bytesArray = bytes.ToArray();
+
+            if (!bytes.SequenceEqual(ExpectedV4Bytes) && !V4BytesFoundOnEwModel.Any(x => x.SequenceEqual(bytesArray)))

```

Comment:
Can't `ExpectedV4Bytes` be moved into the `V4BytesFoundOnEwModel` array and this just reduces to:
`if (!V4BytesFoundOnEwModel.Any(x => x.SequenceEqual(bytesArray)))`?

Reply:
Maybe the check can be put in its own function as well: e.g. `IsVersion4(bytes)`

Reply:
I tried updating it now, please check if its improved 👍